ccm_web/server/src/canvas/canvas.errors.ts

```diff
@@ -0,0 +1,22 @@
+export class CanvasOAuthAPIError extends Error {
+  date: Date
+  public name = 'CanvasOAuthError'
```

Comment:
Minor: Do you mean to name the public error name as `CanvasOAuthAPIError` instead of `CanvasOAuthError`? either way it is fine, but felt class name and public name could be same

Reply:
Oh, yeah, I'll fix that, must have missed it.

Reply:
Done, see https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/135/commits/9d00fe543c9f4081e6494e63e3d3faf71b54b2de